kitsune/sumo/static/sumo/js/instant_search.js

```diff
@@ -93,6 +101,8 @@
       });
     }
 
+    _gaq.push(['_trackEvent', 'Instant Search', 'Search', $this.data('href')]);
```

Comment:
We need to track these as page views that match Google Analytics' search tracking so we can use all the search metrics.